code/src/App.js

```diff
@@ -1,9 +1,71 @@
-import React from 'react';
+import React, { useState } from 'react';
+
+import Header from 'components/Header';
+import { RecieverNameInput } from 'components/Recievernameinput';
+import { ConsiderationInput } from 'components/Considerationinput';
+import { TextAreaInput } from 'components/Textareainput';
+import { ActionInput } from 'components/Actioninput';
+import { UserNameInput } from 'components/Usernameinput';
+import { Summary } from 'components/summary';
+import { Button } from 'components/button';
+import { Email } from 'components/EmailInput';
+import { Checkbox } from 'components/Checkbox';
 
 export const App = () => {
+  const [counter, setCounter] = useState(0);
+  const [recieverName, setRecieverName] = useState('');
+  const [consideration, setConsideration] = useState('');
+  const [textArea, setTextArea] = useState('');
+  const [action, setAction] = useState('');
+  const [userName, setUserName] = useState('');
+
+  const handleStepIncrease = (shouldIncrease) => {
+    if (shouldIncrease && counter < 1) {
+      setCounter(counter + 1)
+    } else if (!shouldIncrease && counter > 0) {
+      setCounter(counter - 1)
+    }
+  }
+
   return (
-    <div>
-      Find me in src/app.js!
+    <div className="outer-wrapper">
+      <div className="inner-wrapper">
+        {/* Where we gather the message */}
+        {counter === 0 && (
+          <div className="form-wrapper">
+            <Header />
+            <RecieverNameInput recieverName={recieverName} setRecieverName={setRecieverName} />
```

Comment:
I noticed that the required attribute doesn't seem to be working. I didn’t have time to try this out so I’m not 100% sure it’ll do the trick, but I think that adding a <form> tag that wraps all the form elements should fix it. (Note that if you would want to try this out, any other <form> tags in the code would have to be removed since they can’t be nested 🙂)